purefa/pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+requires = [
+    "hatchling>=0.13.0",
+    "setuptools; python_version < '3.0'",
```

Comment:
Won't be necessary since this integration only supports python 3.
```suggestion
```

Reply:
You can also remove `requirements.in` and `MANIFEST.in`.

Reply:
Ack